Events/PinwheelInputAmountPayload.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  PinwheelInputAmountPayload.swift
+//  PinwheelSDK
+//
+//  Created by Robby Abaya on 2/12/21.
+//
+
+import Foundation
+
+public struct PinwheelInputAmountPayload: PinwheelEventPayload {
```

Comment:
This is identical to PinwheelAmountPayload so I'd chosen to just have the one type. I can see the argument to be made to have a separate one like this so I'll leave it up to you - just wanted to mention for thought.

Reply:
Ah yes, I meant to change this and forgot. Good catch!